### PR TITLE
[WIP] create recipe for acesinger

### PR DESCRIPTION
**Pull request by linyueqian**
## What?

create recipe for acesinger

### CODE REVIEW

File: egs2/acesinger/svs1/local/midi-note.scp [hunk 1]
```diff
@@ -0,0 +1,108 @@
+127	G9
```
Comment (ftshijt) [inline egs2/acesinger/svs1/local/midi-note.scp:1]: You may symlink this file from other recipes

**Comment by ftshijt** [comment]
Also please add an entry in readme (egs2/README.md

**Comment by ftshijt** [comment]
LGTM! will merge it after CI passed

File: egs2/opencpop/svs1/local/midi-note.scp [hunk 2]
```diff
@@ -105,3 +105,4 @@
 23	B0
 22	A#0/Bb0
 21	A0
+0	rest
```
Comment (sw005320) [inline egs2/opencpop/svs1/local/midi-note.scp:108]: is it OK to be included in this PR?

Comment (linyueqian) [reply egs2/opencpop/svs1/local/midi-note.scp:108]: I added a `0 rest` entry to simplify the MIDI mapping. The original opencpop recipe handles 'rest' by mapping it to 0 (`midis = [midi_mapping[note] if note != 'rest' else 0 for note in midis]`). Including `0 rest` directly in the mapping file makes audio segmentation of the ACESinger easier, especially when the transcription includes 'rest' notations. I believe in this way, we can eliminate the need for special handling in the code.

Comment (sw005320) [reply egs2/opencpop/svs1/local/midi-note.scp:108]: Got it.
Thanks for the note.

**Comment by sw005320** [comment]
Please add `README.md` to include the results and pre-trained model links in a follow-up PR.

**Pull request merged**